be/src/common/daemon.cpp

```diff
@@ -394,6 +394,7 @@ static void init_doris_metrics(const std::vector<StorePath>& store_paths) {
 void signal_handler(int signal) {
     if (signal == SIGINT || signal == SIGTERM) {
         k_doris_exit = true;
+        LOG(INFO) << "doris start to exit";
```

Comment:
also need to output to stderr, then it will be redirect to be.out.
we could get start or stop info from be.out.